### PR TITLE
[Work In Progress] Trash hint on limit hit

### PR DESCRIPTION
**Pull request by benschac**
#1281 

update / feature request

**What kind of change does this PR introduce?**
code change -- will add a tooltip based on users sandbox count

https://github.com/codesandbox/codesandbox-client/issues/1281

**What is the current behavior?**
A user will hit there limit and not know why.

- onHover: x sandboxes in trash 

**What is the new behavior?**
- A tooltip that tells you how many sandboxes are in your trash.

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Sidebar/TrashItem/index.js [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
+            isOver && canDrop ? { backgroundColor: 'rgba(0, 0, 0, 0.3)' } : {}
+          }
+        />
+      </Tooltip>
     </div>
   );
 
 export const entryTarget = {
-  drop: (props, monitor) => {
+  drop: monitor => {
```
Comment (benschac) [inline packages/app/src/app/pages/Dashboard/Sidebar/TrashItem/index.js]: N/A - just an argument that wasn't being used.

Comment (MichaelDeBoey) [reply packages/app/src/app/pages/Dashboard/Sidebar/TrashItem/index.js]: Don't forget to change the call signature too then

**Comment by SaraVieira** [comment]
Hey @benschac !

Thank your for contributing, need any help? :)

**Comment by benschac** [comment]
Hello! Yes! I am having trouble getting my local environment running. Just gave it another crack. `rm -rf` 'ed and re-installed.

Using node v. 12.3.0 
yarn 1.16

this is the first error I'll get: 
```
Error running install script for optional dependency: "/Users/benjaminschachter/codesandbox-client/node_modules/fsevents, /Users/benjaminschachter/codesandbox-client/node_modules/gatsby/node_modules/webpack-dev-server/node_modules/fsevents: Command failed.
```

Updated `node-gyp` cleaned the cache from yarn, deleted node-modules. No luck.

I'll get stuff like this: 

```
node-pre-gyp WARN Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v72-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for fsevents@1.2.4 and node@12.3.0 (node-v72 ABI, unknown) (falling back to source compile with node-gyp) 
```

Still google around.  Looking for a solution.

**Comment by benschac** [comment]
I thought it was coming from a Gatsby Plugin on the site. Anyone else having this problem. Looks like `fs-modules` is getting a 404 and causing all sorts of havoc

**Comment by benschac** [comment]
Upgraded my system, no luck. Curious, Y'all have an internal `yarn.lock` file? Maybe that could be the issue? This third party dep is out of sync?

**Comment by benschac** [comment]
Finally got my environment to work. Downgraded to node version 8 :D. Getting build errors, but it's building woooo!

**Comment by benschac** [comment]
```
Failed to compile.

../common/lib/utils/analytics.js
Module not found: Error: Can't resolve '../version' in '/Users/benjaminschachter/codesandbox-client/packages/common/lib/utils'
```

getting closer!

**Comment by CompuIves** [comment]
Interesting. I think everything should work if you merge with master and run `yarn && yarn build:deps` in the root of the project.

**Comment by benschac** [comment]
it does. This was resolved in another PR. tyty

**Comment by SaraVieira** [comment]
Hey!

Thank you so much for your help but last week we removed all the limits for free users so this PR is no longer needed but in a good way 🎉

**Comment by benschac** [comment]
ty. new front-end looks lovely :D

**Pull request closed without merging**